kubernetes/linux/main.sh

```diff
@@ -750,7 +751,11 @@ if [ "${CONTAINER_TYPE}" == "PrometheusSidecar" ]; then
 else
       echo "starting mdsd in main container..."
       # add -T 0xFFFF for full traces
-      mdsd ${MDSD_AAD_MSI_AUTH_ARGS} -e ${MDSD_LOG}/mdsd.err -w ${MDSD_LOG}/mdsd.warn -o ${MDSD_LOG}/mdsd.info -q ${MDSD_LOG}/mdsd.qos 2>>/dev/null &
+      export MDSD_ROLE_PREFIX=/var/run/mdsd-ci/default
+      echo "export MDSD_ROLE_PREFIX=$MDSD_ROLE_PREFIX" >> ~/.bashrc
+      source ~/.bashrc
+      mkdir /var/run/mdsd-ci
+      mdsd ${MDSD_AAD_MSI_AUTH_ARGS} -r ${MDSD_ROLE_PREFIX} -e ${MDSD_LOG}/mdsd.err -w ${MDSD_LOG}/mdsd.warn -o ${MDSD_LOG}/mdsd.info -q ${MDSD_LOG}/mdsd.qos 2>>/dev/null &
```

Comment:
MDSD has default 20KPS. with the syslog, MDSD can throttle on high scale existing customers where they have high scale container log. we should bump this when syslog enabled or have config option to provide if customers if needed.

Reply:
have we exposed this option in some other place currently?